packages/stitcher/src/types.ts

```diff
@@ -1,15 +1,19 @@
 import type { DateTime } from "luxon";
 
-export type InterstitialAssetType = "ad" | "bumper";
-
-export interface InterstitialAsset {
-  url: string;
-  type?: InterstitialAssetType;
-}
-
-export interface Interstitial {
+export type Interstitial = {
```

Comment:
I'm in doubt whether we should store each type string in a kv, this'll waste several bytes on storage.